src/mongocxx/collection.hpp

```diff
@@ -466,6 +466,20 @@ class MONGOCXX_API collection {
     stdx::string_view name() const noexcept;
 
     ///
+    /// Rename this collection.
+    ///
+    /// @param new_name The new name to assign to the collection.
+    /// @param drop_target_before_rename Whether to overwrite any
+    ///   existing collections called new_name. The default is false.
+    ///
+    /// @throws exception::operation if the operation fails.
+    ///
+    /// @see https://docs.mongodb.org/manual/reference/command/renameCollection/
+    ///
+    void rename(stdx::string_view new_name, bool drop_target_before_rename = false);
```

Comment:
I know this predates your changes (and that this was already merged), but I really dislike bool parameters of this form. They make code that uses the API inscrutable.

Is this signature mandated by the CRUD specification? If not, perhaps we would do better with either an enum or two methods, though I'm having difficulty with what the method name should be fore the compound operation.